lesson03/main.js

```diff
@@ -0,0 +1,17 @@
+const ul = document.getElementById("js-target");
+const imgSrcs = ["/img/bookmark.png","/img/messege.png"];
+
+for (let i=0; i < imgSrcs.length; i++){
+  const li = document.createElement("li");
+  const a = document.createElement("a");
+  const img = document.createElement("img");
+
+  a.href = `a${i}.html`;
+  a.textContent = `a${i}`;
```

Comment:
![Screen Shot 2022-11-24 at 9 02 36 pm](https://user-images.githubusercontent.com/86552484/203756115-a476c47e-ce0e-43ec-bfc4-a3f66cd6a8c0.png)

nit:
It might be better to start with the number 1 so that it is the same as question.

```suggestion
a.textContent = `a${i + 1}`;
```

Reply:
Thank you saki-san :）
I fixed my code.
58d8884dedb016e51b6cdfb17c55057383c5d685